include/dsn/cpp/serverlet.h

```diff
@@ -111,6 +111,8 @@ namespace dsn
         template<typename TResponse>
         void reply(dsn_message_t request, const TResponse& resp);
 
+        template<typename TResponse>
```

Comment:
I thought these kinds of hacks are not necessary now - it is best we come up more elegant approaches.